test.sh

```diff
@@ -372,239 +276,76 @@ parse_version_arg() {
+  local action=$1 ; shift
+  local args=$@
+  gojira $action -v ${GOJIRA_SETTINGS[*]} --repo $repo -t $version --host $host $@
 }
 
-prepare_repo() {
-    repo=$1
-    version=$2
-    tmp_repo_name=$repo-`builtin echo $version | sed 's/\//_/g'`
-    ret1=$tmp_dir/$tmp_repo_name
-
-    if [[ "$keep" = 1 ]]; then
-      pushd $tmp_dir/$tmp_repo_name
-        ret2=`grep OPENRESTY_BASE= .travis.yml | cut -d = -f 2`
-        ret3=`grep OPENSSL= .travis.yml | cut -d = -f 2`
-        ret4=`grep LUAROCKS= .travis.yml | cut -d = -f 2`
-      popd
-
-      return
-    fi
-
-    pushd $tmp_dir
-      cp -R $cache_dir/$repo $tmp_repo_name \
-        || show_error "cp failed with: $?"
-      pushd $tmp_repo_name
-        git checkout $version \
-          || { co_exit=$?; rm -rf $tmp_repo_name; show_error "git checkout to '$version' failed with: $co_exit"; }
-        ret2=`grep OPENRESTY_BASE= .travis.yml | cut -d = -f 2`
-        ret3=`grep OPENSSL= .travis.yml | cut -d = -f 2`
-        ret4=`grep LUAROCKS= .travis.yml | cut -d = -f 2`
-      popd
-    popd
-
+has_new_migrations() {
+  local repo=$1
+  local version=$2
+  case $repo in
+    kong-ee)
+      # Strip release/ from tag
+      version=$(builtin echo $version | sed -e "s/release\///")
+
+      if [[ "$version" = "master" ]] || [[ "$version" =~ "/" ]]; then
+        ret1=0
+        return
+      fi
+      ee_semverGT $version 0.34.999 5>/dev/null
+      ret1=$?
+      ;;
+    kong)
+      if [[ "$version" = "next" ]] || [[ "$version" =~ "/" ]]; then
+        ret1=0
+        return
+      fi
+      semverGT $version 0.14.1 5>/dev/null
+      ret1=$?
+      ;;
+  esac
 }
 
-set_env_vars() {
-    pref=$1
-    admin=$2
-    proxy=$3
-    admin_ssl=$4
-    proxy_ssl=$5
-    path=$6
-    openssl_dir=$7
-
-    export KONG_PREFIX="$pref"
-    if grep -q "proxy_listen_ssl" kong.conf.default; then
-        export KONG_ADMIN_LISTEN="$admin"
-        export KONG_PROXY_LISTEN="$proxy"
-        export KONG_ADMIN_LISTEN_SSL="$admin_ssl"
-        export KONG_PROXY_LISTEN_SSL="$proxy_ssl"
-    else
-        export KONG_ADMIN_LISTEN="$admin, $admin_ssl ssl"
-        export KONG_PROXY_LISTEN="$proxy, $proxy_ssl ssl"
-    fi
-    export PATH=$path
-    export OPENSSL_DIR=$openssl_dir
-}
+install_kong() {
+    local repo=$1
+    local version=$2
+    local image
 
-using_kong_node() {
-  if [[ "$1" = "1" ]]
-  then
-    set_env_vars $prefix $ADMIN_LISTEN $PROXY_LISTEN $ADMIN_LISTEN_SSL $PROXY_LISTEN_SSL \
-      "$OPENSSL_INSTALL_1/bin:$OPENRESTY_INSTALL_1/bin:$OPENRESTY_INSTALL_1/nginx/sbin:$LUAROCKS_INSTALL_1/bin:$orig_path" \
-      $OPENSSL_INSTALL_1
-  else
-    set_env_vars $prefix_2 $ADMIN_LISTEN_2 $PROXY_LISTEN_2 $ADMIN_LISTEN_SSL_2 $PROXY_LISTEN_SSL_2 \
-      "$OPENSSL_INSTALL_2/bin:$OPENRESTY_INSTALL_2/bin:$OPENRESTY_INSTALL_2/nginx/sbin:$LUAROCKS_INSTALL_2/bin:$orig_path" \
-      $OPENSSL_INSTALL_2
-  fi
-}
+    if ! [[ "$rebuild" = "1" ]]; then
+      image=$(gojira snapshot? --repo $repo -t $version)
```

Comment:
This (as well as other steps) assume that there is a `gojira` executable in the path. It seems that the intention is using `./kong-gojira/gojira.sh` instead?

A quick fix for this could be adding this function:

```
gojira() {
  ./kong-gojira/gojira.sh "$@"
}
```

Reply:
resolved in 3f24583